src/components/Sum.jsx

```diff
@@ -1 +1,5 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (
```

Comment:
Maybe it is better touse deafult props here
And also, don't forget to add prop  types